soc/arm/Kconfig

```diff
@@ -57,6 +57,14 @@ config NRF_SPU_RAM_REGION_SIZE
 	  RAM region size for the NRF_SPU peripheral
 endif
 
+if HAS_HW_NRF_ACL
```

Comment:
Can you place this in nRF sub directory?
Rationale:
This file should _ideally_ be for ARM Core peripheral and their implementation specific variants
- MPU (so ARM MPU, NXP MPU, etc.)
- SAU (so ARM SAU and all IDAUs)

Another option is to clearly deprecate this file, so let's not pollute it even more with SoC-particular definitions.

Reply:
I moved it to nordic_nrf/Kconfig.peripherals. Tell me if you think it should be somewhere else.